amr-wind/setup/init.cpp

```diff
@@ -10,6 +10,38 @@
 
 using namespace amrex;
 
+void incflo::CheckAndSetUpDryRun()
```

Comment:
hmmm I am not sure the order of operations in the parmparse. Could you check with, e.g. having 
```
incflo.initial_iterations = 3
incflo.do_initial_proj =1
incflo.dry_run =1
```
in the input file? Do those values get reset by this function?

Reply:
I checked by doing dry runs with abl_godunov. Those values get reset with this function. Regardless of the arguments in the input file or the command line, the initial iterations get set to 0 and the initial projection gets skipped. That's the intention. (If someone wanted those to happen, it's still possible to do the same thing, just not using the dry run option)

Reply:
I also made sure that it works with abl_bndry_input_native; plotfile written successfully.

Reply:
awesome. Thanks!